blog/on_retros/README.md

```diff
@@ -0,0 +1,272 @@
+
+This is the exact kind of "Art of War" wisdom that I love because in some ways it's like "well, duh" but also I find the ideal so well articulated that I keep coming back to it and applying it as a problem-solving framework.
+
+In software terms this might be
+1) Have you identified the REAL problem?
+2) Have you identified the REAL solution (or at least a good solution)?
+3) Have you actually executed the solution well?
+
+Any of these can fail, and its crucial to be able to identify what step you are failing on. Too often I see teams do 1 & 2 correctly, but lack the experience as a team to execute on the solution. They then abandon the solution entirely instead of working to build the competency needed to execute it.
+
+Being at least "pretty good" at this kind of problem-solving is essential to running successful retros.
+
+Here is a rough outline of what I'd like to cover:
+1) Why retros are so important
+2) What makes a retro "good" and what makes one "bad"
+3) Some success stories
+4) A collection of common problems I've seen teams encounter and some possible retro driven solutions (read: patterns!).
+
+And finally, as a massive disclaimer, these are all ideas and recommendations from my own personal experience. What works for me may not work for you. I'm certain that in another year/month/tomorrow, I will learn something new and change all my recommendations here. Feel free to take and try anything you like but just remember that there is no "one true way" and agile/retros are all about learning and trying new things.
+
+## Why Retros?
+[Webster's Dictionary defines](https://www.youtube.com/watch?v=s93EZfNvOyw&ab_channel=SCProductions) retros as
+
+>a meeting to look back over an iteration, release, or project, specifically to discuss what worked well, what could be improved, and most importantly, how to translate the lessons learned into actionable change. Retrospectives are a forum for the team to improve upon their process. They’re an integral part of Scrum and Extreme Programming (XP).
+>
+>Retrospectives directly address one of the twelve principles of Agile: At regular intervals, the team reflects on how to become more effective, then tunes and adjusts its behavior accordingly.
+>
+>This can result in: improved productivity, better capability, higher product quality, and happier team members.
+
+[*actual source](https://www.intelliware.com/agile-retrospectives/)
+
+It sounds innocuous, but it's actually "the most important" thing you can do to improve your teams. If I had to pick one single process to bring to any team, its retros. Everything else is emergent from retros.
+
+Retros matter because:
+1) they provide a simple framework to continuously and iteratively improve you team, your work, and ideally, your life.
+2) when you improve your own life, you tend to provide better outcomes for the business (happy engineers => better outcomes, less turnover, easier recruiting, more learning, etc.)
```

Comment:
Interesting to think on how it will improve your life. What says you to the developer who say "it is just a job"

Reply:
added a little blurb about this in section below